src/main/java/ch/njol/skript/expressions/ExprArgument.java

```diff
@@ -86,8 +87,8 @@ public class ExprArgument extends SimpleExpression<Object> {
 	@SuppressWarnings("unchecked")
 	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
 		boolean scriptCommand = getParser().isCurrentEvent(ScriptCommandEvent.class);
-		if (!scriptCommand && !getParser().isCurrentEvent(PlayerCommandPreprocessEvent.class, ServerCommandEvent.class)) {
-			Skript.error("The 'argument' expression can only be used in a script command or command event");
+		if (!scriptCommand && !getParser().isCurrentEvent(PlayerCommandPreprocessEvent.class, ServerCommandEvent.class, UnknownCommandEvent.class)) {
```

Comment:
This is a Paper exclusive event. Needs a Skript.classExists method check. Typically for init methods, we put a private static final boolean field in the class that defines if the class exists for usage.

Reply:
just to double check this is the format is needs to follow
![image](https://user-images.githubusercontent.com/81952476/210126656-e7d32470-a7b2-4b0c-b186-288e24fb7b9f.png)